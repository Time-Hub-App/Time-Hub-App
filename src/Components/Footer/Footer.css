footer h3:hover {
  cursor: pointer;
  text-decoration: underline;
}
