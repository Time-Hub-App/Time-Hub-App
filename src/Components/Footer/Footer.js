import { useHistory } from 'react-router-dom';
import './Footer.css';

export default function Footer() {
  const history = useHistory();
  return (
    <footer>
      <h3 onClick={() => history.push('/about')}>About the Creators</h3>
    </footer>
  );
}
