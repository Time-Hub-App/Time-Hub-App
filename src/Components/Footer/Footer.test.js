import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Footer from './Footer';

it('should render a footer', () => {
  const { container } = render(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
