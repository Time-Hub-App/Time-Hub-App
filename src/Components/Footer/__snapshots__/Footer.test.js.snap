// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a footer 1`] = `
<div>
  <footer>
    <a
      class="footer-link"
      href="/https://github.com/Time-Hub-App"
    >
      TimeHub GitHub
    </a>
  </footer>
</div>
`;
